Source/Project64/UserInterface/SettingsConfig.cpp

```diff
@@ -112,9 +112,10 @@ LRESULT	CSettingConfig::OnInitDialog(UINT /*uMsg*/, WPARAM /*wParam*/, LPARAM /*
     {
         if (g_Settings->LoadBool(Setting_RdbEditor))
         {
-            SetWindowText(stdstr_f("%ws ** RDB Edit Mode **", GS(OPTIONS_TITLE)).c_str());
+            ::SetWindowTextW(m_hWnd, stdwstr_f(L"%ws ** RDB Edit Mode **", wGS(OPTIONS_TITLE).c_str() ).c_str());

```

Comment:
it probaly should be SetWindowTextW(stdstr_f("%s ** RDB Edit Mode **", GS(OPTIONS_TITLE)).toUtf16().c_str());